modelstore/models/xgboost.py

```diff
@@ -61,6 +62,9 @@ def _get_functions(self, **kwargs) -> list:
         ]
 
     def _get_params(self, **kwargs) -> dict:
+        if type(kwargs["model"]).__name__ == "Booster":
```

Comment:
For consistency, can you use `isinstance()` here too?

```suggestion
        import xgboost as xgb
        if isinstance(kwargs.get("model"), xgb.Booster):
```